docs/release-notes/NuGet-5.11.md

```diff
@@ -0,0 +1,50 @@
+---
+title: NuGet 5.11 Release Notes
+description: Release notes for NuGet 5.11 including new features, bug fixes, and DCRs.
+author: erdembayar
+ms.author: eryondon
+ms.date: 8/10/2021
+ms.topic: conceptual
+---
+
+# NuGet 5.11 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.11.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.11](https://visualstudio.microsoft.com/downloads/) | [5.0.400](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+  
+> [!NOTE]
+> Visual Studio 16.11, MSBuild 16.11, and .NET 5.0.400+ requires NuGet.exe 5.11 or later.
+
+## Summary: What's New in 5.11
```

Comment:
Were there no new features, enhancements, or significant performance improvements in this release? 

cc @nkolev92 @aortiz-msft

Reply:
No, 5.11 was a stability release with a *very* high bar.

Reply:
I'm not sure why but there are very few commits for 5.11
https://github.com/NuGet/NuGet.Client/compare/5.10.0.7240...5.11.0.17

Is this one qualify for release notes: https://github.com/NuGet/Home/issues/10878

Reply:
Thanks for the clarification Nikolche, just wanted to make sure 👍